code/modules/admin/verbs/SDQL2/SDQL_2.dm

```diff
@@ -304,19 +304,19 @@
 		if(op != "")
 			switch(op)
 				if("+")
-					result += val
+					result = (result + val)
```

Comment:
I'm fucking sick and tired of this shit where people touch code they don't understand.

By adding the comment, you imply that every bit of code thats the same without the same comment is safe to change, but thats not true.

These landmines are important for teaching new coders not to fuck with shit they don't understand.

When you see something and go "why is it doing it that way". the answer is not to go "fuck it, it seems to still work when i change it" the answer is to leave it the fuck alone until it doing it that way is actually an issue.

How many god damn bugs have we had this last year from non-changes like that. I'm sick of them.